### PR TITLE
Fix older posts button positioning after loading posts.

### PR DESCRIPTION
**Pull request by mendezcode**
This PR fixes one of the issues reported in #2179

There is currently another issue after clicking the "Older Posts"
button, which causes the posts to overlap each other. This will be fixed
in another PR.

### CODE REVIEW

**Comment by mendezcode** [comment]
The button was absolutely positioned and it was making it overlap with the posts.
My solution involves floating the button and making it relatively positioned.
There's a `top: 100%` property too, which I'm overriding when `.infinity-success` is added to the body.

File: altofocus/style.css [hunk 1]
```diff
@@ -3189,10 +3189,16 @@ body {
   clear: both;
   display: block;
   padding: 1.75em 10%;
-  position: absolute;
   text-align: center;
   width: 100%;
   z-index: 1;
+  position: relative;
```
Comment (kjellr) [inline altofocus/style.css:3195]: This needs to be changed in `altofocus/assets/stylesheets/shared/_infinitescroll.scss`, and then compiled here.

**Comment by mendezcode** [comment]
Since there is no build tooling integrated into Altofocus, I installed Sass globally with `npm install --global sass`, and then compiled with `sass --no-source-map assets/stylesheets`. 

I then moved the `assets/stylesheets/style.css` to the theme's root directory.

As can be noticed in the [changed files](https://github.com/Automattic/themes/pull/2264/files), there are a lot of changes added, which make me a bit hesitant about pushing this change forward due to the amount of changes in the compiled sources.

@allancole since you worked some time ago with Altofocus, do you mind sharing the app/tool (and its configuration) used to compile the Sass for the theme?

Also, as an alternative method, you could also add [the changes](https://github.com/Automattic/themes/files/4933483/altofocus.diff.zip) and provide me with the compiled diiff, so I can use this as a patch for this PR:

**Comment by danieldudzic** [comment]
@kjellr I have investigated the reason for the new posts overlap. This is caused by a failure inside of the re-layout function.
 
A list of freshly loaded posts should be set in:

https://github.com/Automattic/themes/blob/9306ee271fba16605ec673966ffaa6bea4b34539/altofocus/inc/jetpack.php#L180

But for some reason, when the re-layout function runs that global variable is not set.

So the whole thing stops on this check:

https://github.com/Automattic/themes/blob/9306ee271fba16605ec673966ffaa6bea4b34539/altofocus/assets/js/grid.js#L103

Tried debugging further but got stuck a bit.

Perhaps @jeffikus will have an idea as to why this is happening.

**Comment by allancole** [comment]
Did some testing with this and I’m wondering if this is ultimately a Jetpack issue. When I test locally, and look at the console logs for `loadedPosts`, it properly returns an array of post ids and gathers them together as expected. But on wpcom it returns as an `undefined` variable. From what I can tell, this is happening because Jetpack on dotcom isn’t rendering  `altofocus_infinite_scroll_render` ([here](https://github.com/Automattic/themes/blob/b9983eb8217d12a5f07f9138a624e1f8dd14832a/altofocus/inc/jetpack.php#L22)) the same way it does locally with the latest version of the Jetpack plugin which shouldn’t be the case.

**Comment by danieldudzic** [comment]
@allancole Not sure why this isn't working for you on WP.com - I have asked @jffng to test it on his sandbox and he confirmed, that it resolves the issues for him.

Since it also works fine on my sandbox - I'll merge and deploy the fix.

**Pull request merged**